sample/test-sample/frontend/src/main/Main.js

```diff
@@ -33,6 +35,16 @@ export const Main = () => {
       <Route exact path="/new/project" component={withErrorBoundary(withCapabilities(NewProjectView))} />
       <Route exact path="/upload/project" component={withErrorBoundary(withCapabilities(UploadProjectView))} />
       <Route exact path="/projects" component={withErrorBoundary(withCapabilities(ProjectsView))} />
+      <Route
+        exact
+        path="/projects/:projectId/modelers"
+        component={withErrorBoundary(withCapabilities(withProject(ModelersView)))}
```

Comment:
Do not rely on the ProjectContext for useProject, you don't need it. Do you really need to fetch the capabilities too?